docker/receiver.py

```diff
@@ -60,22 +60,24 @@ def runfunc(self):
         try:
             output = subprocess.check_output(application, stderr=subprocess.STDOUT, shell=True, cwd=folder)
             status = 'OK'
-            logging.info("Finished running job.")
         except subprocess.CalledProcessError as e:
-            logging.exception("Error running job.", e)
+            logging.exception("Error running job.")
             output = e.output
             status = 'ERROR'
         except Exception as e:
-            logging.exception("Error running job.", e)
+            logging.exception("Error running job.")
             output = str(e)
             status = 'ERROR'
 
+        logging.info("Finished running job with status " + status)
+        logging.info(output)
+
         try:
             with open(os.path.join(folder, 'rabbitmq.out'), 'w') as out:
                 out.write(str(output) + "\n")
                 out.write(status + "\n")
-        except Exception as e:
-            logging.exception("Error writing status.", e)
+        except Exception:
+            logging.exception("Error writing status.")
```

Comment:
I'd rather you put the `e` back in here. Otherwise, user has not idea what the error was.

Reply:
Got it, thanks for clarifying! In that case, this is good to go.